packages/upload-client/src/car.js

```diff
@@ -7,7 +7,7 @@ import varint from 'varint'
  */
 
 /** Byte length of a CBOR encoded CAR header with zero roots. */
-const NO_ROOTS_HEADER_LENGTH = 17
+const NO_ROOTS_HEADER_LENGTH = 18
```

Comment:
🤦 FML this was wrong. Either it was wrong the whole time or `@ipld/car` started encoding the header differently.